### PR TITLE
London Class 8 - Maryam Mohsenpour - JS/Core 2 - WEEK 4

### PR DESCRIPTION
**Pull request by bita2019**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name: Maryam Mohsenpour
- Your City:London
- Your Slack Name: Maryam 

# Homework Details

- Module: JS/2
- Week: 4

# Notes

- What did you find easy? Array gets much easier as we use it more and more.

- What did you find hard? problem solving

- What do you still not understand? -

- Any other notes?

### CODE REVIEW

**Comment by bita2019** [comment]
@DiegoJunior0  hey, this is 2 of 3 exercises.

File: 1-mandatory/1-alarmclock/alarmclock.js [hunk 1]
```diff
@@ -1,4 +1,21 @@
...
+  let myDate = new Date("2017-01-01 00:00:00");
+  myDate.setSeconds(MyInputAlarmSet.value);
+  MyTimeremaining.innerHTML =
+    "Time Remaining : " + myDate.toString().substring(19, 24);
+  let myInterval = setInterval(() => {
+    myDate.setSeconds(myDate.getSeconds() - 1);
+    MyTimeremaining.innerHTML =
+      "Time Remaining" + myDate.toString().substring(19, 24);
+    if (myDate.toString().substring(19, 24) === "00:00") {
+      playAlarm();
```
Comment (DiegoJunior0) [inline 1-mandatory/1-alarmclock/alarmclock.js:14]: An easier way of getting the seconds would be `myDate.getSeconds()`
So you dont have to extract it from the string

Comment (DiegoJunior0) [reply 1-mandatory/1-alarmclock/alarmclock.js:14]: Good idea using a date though!

**Comment by DiegoJunior0** [comment]
Good work Maryam!

**Comment by bita2019** [comment]
thanks Henry

File: 1-mandatory/1-alarmclock/alarmclock.js (hunk 1 above)
Comment (bita2019) [reply 1-mandatory/1-alarmclock/alarmclock.js:14]: thanks

**Pull request closed without merging**